### PR TITLE
M69

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

**Comment by saghul** [comment]
@paweldomas Can you PTAL?

File: ios/RCTWebRTC/RTCVideoViewManager.m [hunk 1]
```diff
@@ -147,10 +147,7 @@ - (void)layoutSubviews {
   CGFloat width = _videoSize.width, height = _videoSize.height;
   CGRect newValue;
   if (width <= 0 || height <= 0) {
-    newValue.origin.x = 0;
-    newValue.origin.y = 0;
-    newValue.size.width = 0;
-    newValue.size.height = 0;
+    newValue = self.bounds;
```
Comment (paweldomas) [inline ios/RCTWebRTC/RTCVideoViewManager.m:150]: Question about this change: can you explain what problem is it addressing ?

Comment (saghul) [reply ios/RCTWebRTC/RTCVideoViewManager.m:150]: I say video go black sometimes and I suspected the view was there but with size 0 so I thought it's reasonable to start with the bounds of the enclosing view until we have a frame to take the measurements.

**Review by paweldomas — APPROVED** [review]
Approved without testing

**Pull request merged**